cloudinit/sources/DataSourceAeza.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (C) 2024 Aeza.net.
+#
+# Author: Egor Ternovoy <cofob@riseup.net>
+#
+# This file is part of cloud-init. See LICENSE file for license information.
+
+from cloudinit import sources, util
+import cloudinit.sources.helpers.aeza as aeza_helper
+
+BASE_URL_V1 = "http://77.221.156.49/v1/cloudinit"
+
+BUILTIN_DS_CONFIG = {
+    "retries": 60,
+    "timeout": 2,
+    "wait_retry": 2,
+    "metadata_url": BASE_URL_V1 + "/{id}/meta-data",
+    "userdata_url": BASE_URL_V1 + "/{id}/user-data",
+    "vendordata_url": BASE_URL_V1 + "/{id}/vendor-data",
+}
+
+class DataSourceAeza(sources.DataSource):
+
+    dsname = "Aeza"
+
+    def __init__(self, sys_cfg, distro, paths, ud_proc=None):
+        sources.DataSource.__init__(self, sys_cfg, distro, paths, ud_proc)
+        self.metadata = {}
+        self.vendordata_raw = None
+        self.userdata_raw = ""
+
+        self.ds_cfg = util.mergemanydict(
+            [
+                util.get_cfg_by_path(sys_cfg, ["datasource", "Aeza"], {}),
+                BUILTIN_DS_CONFIG,
+            ]
+        )
+
```

Comment:
super().__init__ already sets these up no need to redefine here.
```suggestion
```

Reply:
Fixed in 4350dc221feb40ba2e277706e890126f92cee282
`self.ds_cfg` merge had to remain, otherwise the tests didn't work.

Reply:
Thank you. My mistake on inclusion there. I overlooked the BUILTIN_DS_CONFIG you are merging as well.